include/seqan/rna_io/ebpseq_read_write.h

```diff
@@ -451,7 +451,7 @@ writeHeader(TTarget & target, RnaHeader const & header, RnaIOContext & context,
 
 inline void createPseudoHeader(RnaHeader & header, std::vector<RnaRecord> & records)
 {
-    std::size_t numFix{}, numBpp{};
+    bool hasFixed = false, hasBppMatr = false;
     for (unsigned idx = 0; idx < length(records); ++idx)
```

Comment:
please make idx the same type as length(records) to prevent shortening (we will have warnings for this in seqan3).